attestor/command/cli.go

```diff
@@ -114,7 +114,7 @@ func init() {
 
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
-		fmt.Println(err)
+		fmt.Fprintln(os.Stdout, err) // can this just be stderr?
```

Comment:
Yes, I think this should be `os.Stderr`

Reply:
Is this something that any scorecard attestor users might be relying on?